From 837fa2cdcc36cd7ab400a40696cb4e419cff6281 Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Wed, 11 Jan 2017 00:55:32 +1300
Subject: [PATCH] DAPI-330: Improved over-provisioning logic to take into
 account the currently used amounts DAPI-334: dapi can miscalculate disk and
 ram requirements when a disk or ram arg differs from pkg

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 6ab0d6b..5659015 100644
--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "assert-plus": "1.0.0",
     "async": "1.5.2",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#2276a6b2879e3ea7c86d5b0a476361db0ee35b03",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#921a286254ef3cf678e34933a15417941da0e7de",
     "deep-equal": "1.0.1",
     "dox": "0.4.1",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
-- 
2.21.0

